front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/StrategyController.groovy

```diff
@@ -55,7 +56,7 @@ class StrategyController {
         pipelineStrategyDAO.getPipelinesByApplication(application)
     }
 
-    @PostAuthorize("hasPermission(filterObject.application, 'APPLICATION', 'READ')")
+    @PostFilter("hasPermission(filterObject.application, 'APPLICATION', 'READ')")
```

Comment:
Just for my understanding, does `PostFilter` run the function and then do the authZ check? Or the reverse?